test/e2e/connection-alice-to-bob.spec.ts

```diff
@@ -0,0 +1,218 @@
+import { expect, test } from '@playwright/test'
+import { checkEmails, extractInvite, extractPin, findNewAdminEmail } from './helpers/smtpEmails'
+
+test.describe('Connection from Alice to Bob', () => {
+  let context: any
+  let page: any
+  let adminEmailId: string
+  let invite: string | null
+  let pinForBob: string | null
+  let pinForAlice: string | null
```

Comment:
quite a few lets, wonder if they are all needed

Reply:
` let context: any // shared between all pages
  let page: any 
  let adminEmailId: string // to exclude this when looking for Alice's email containing PIN
  let invite: string | null // for Bob -> we extract in one step and use in another so saving it here 
  let pinForBob: string | null // same as above 
  let pinForAlice: string | null // also extracted in one step and used in another - so saving in this var`